lib/menu.rb

```diff
@@ -0,0 +1,23 @@
+class Menu 
+    attr_reader :dishes
+
+    def initialize
+        @dishes = {
+            "Pizza" => 5,
+            "Pasta" => 8,
+            "Bruschetta" => 3,
+            "Burrata" => 4.5,
+            "Lasagne" => 10,
+            "Gelato" => 2.5,
+            "Tiramisu" => 4.99
+        }
+    end
+
+    def show_menu
+        @dishes.map { |key, value| "#{key}, £#{value}"}
```

Comment:
Maybe could have been more descriptive with the variable names in 'map' e.g. | menu_item, price | but think this still does exactly the right job...:)